Resources/Lang/en_US.json

```diff
@@ -540,6 +541,7 @@
   "BerserkerInfo": "Kill to increase your level",
   "EvilMiniInfo": "No one can hurt you until you grow up",
   "BlackmailerInfo": "Silence other players",
+  "InstigatorInfo": "Sow discord among the crewmates",
```

Comment:
Sow? I'm so lost loool

Reply:
Is it not correct english? haha. How would you say it? Translator says it's fine.

Reply:
sow is correct, but simply do "Cause discord among the crewmates" imo